jha/src/views/enrolment/DocumentsPage.vue

```diff
@@ -9,12 +9,12 @@
     </div>
     <PageContent :deltaHeight='pageContentDeltaHeight'>
       <main class="container pt-3 pt-sm-5 mb-3">
-        <h1>Upload Supporting Documents</h1>
+        <h1>Upload supporting documents</h1>
         <hr class="mt-0"/>
         <div class="row">
           <div class="col-md-8">
             <div>
-              <p class="font-weight-bold">Upload your Canada Revenue Agency Notice of Assessment or Notice of Reassessment for {{selectedNOAYear}}</p>
+              <p class="font-weight-bold">Upload your Canada Revenue Agency NOA or NORA for {{selectedNOAYear}}</p>
```

Comment:
I'll be honest, I might not know what NORA meant in this context if not for the original code version on the left. Is it possible to keep it, eg. Notice of Reassessment (NORA) like happened with CUAET earlier?

Reply:
At this point they will have already seen `Notice of Assessment (NOA) and Notice of Reassessment (NORA)` once